<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibrar</class>
 <widget class="QDialog" name="Calibrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibrar sensores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgs/imgs.qrc">
    <normaloff>:/imgs/utilities.png</normaloff>:/imgs/utilities.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Porta:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="ip_real">
         <property name="text">
          <string>10.13.99.69</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="porta_real">
         <property name="text">
          <string>20081</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Escrita&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="canal_esc">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tres">
       <property name="text">
        <string>Escrever 3V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dois">
       <property name="text">
        <string>Escrever 2V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="um">
       <property name="text">
        <string>Escrever 1V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zero">
       <property name="text">
        <string>Escrever 0V</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>43</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Leitura&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>43</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sensor Tanque 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="canal_leit_t1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Canal 0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 7</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="nivel_t1">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sensor Tanque 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="canal_leit_t2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Canal 0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canal 7</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="nivel_t2">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="iniciar">
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Parar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fechar">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imgs/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fechar</sender>
   <signal>clicked()</signal>
   <receiver>Calibrar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
